### PR TITLE
ONC-42 is completed in possibly the grossest way possible

### PR DESCRIPTION
**Pull request by ApplebaumIan**
We don't have to go through funky steps anymore. Our ChatGPT library is hosted on https://github.com/Civic-Interactions-Lab/chatgpt-api/tree/build

package.json update
```json
...
 "dependencies": {
    "chatgpt": "Civic-Interactions-Lab/chatgpt-api#build"
  }
```

### CODE REVIEW

File: package.json
Comment (IshanAggarwal125) [inline package.json:1]: Looks good to me.

**Review by IshanAggarwal125 — APPROVED** [review]
Looks good to me.

**Review by jbbernardin — APPROVED** [review]
I can't tell from the pull request itself whether these changes work without looking at the chatgpt library you made, but since we saw this working yesterday I trust that it works fine.

**Comment by ApplebaumIan** [comment]
@jbbernardin and @IshanAggarwal125 I think im going to merge #1 first and may request another review of this because I did have trouble with this in lab.

**Comment by ApplebaumIan** [comment]
Tested in codespaces. I think this is gtg.

**Pull request merged**